packages/design-system/src/components/IconButton/__tests__/IconButton.spec.tsx

```diff
@@ -0,0 +1,29 @@
+import IconButton from "..";
+import { IconButtonObject } from "./IconButton.screen-object";
+
+describe(IconButton.name, () => {
+  it("calls the onClick prop functon when IconButton clicked", () => {
+    const onClick = jest.fn();
+    const screen = new IconButtonObject().render({
+      onClick,
+    });
+
+    screen.clickIconButton();
+    expect(onClick).toHaveBeenCalled();
+  });
+  it("doesn't allow to click when IconButton disabled", () => {
+    const onClick = jest.fn();
+    const screen = new IconButtonObject().render({
+      onClick,
+      disabled: true,
+    });
+
+    screen.clickIconButton();
+    expect(onClick).not.toHaveBeenCalled();
+  });
+  it("dislays icon", () => {
```

Comment:
```suggestion
  it("displays icon", () => {
```
The same mistake is in button test 😉